db/index.js

```diff
@@ -56,8 +64,29 @@ const getCharitiesById = function(id, callback) {
   });
 };
 
+
+// db.connectAsync()
+//   .then(() => {
+//     console.log(`Connected to MySQL Charities!`)
+//   })
+//   .then(() => {
+//     db.queryAsync(`CREATE DATABASE IF NOT EXISTS ${database}`)
+//   })
+//   .then(() => {
+//     db.queryAsync(`USE ${database}`)
+//   })
+//   .then(() => {
+//     conf(db)
+//   })
+  // .then(() => {
+  //   db.queryAsync(`INSERT INTO charity (id, name, description) VALUES ('1','Parker-Erdman','Sunt quod autem et et et distinctio. Aut est eum omnis mollitia explicabo quia nulla voluptatibus. Amet consequatur saepe unde iure. Corporis nam doloribus nihil quas ut consequatur. Aliquam perferendis in debitis reprehenderit dolorum.'),
+  //   [...]
+  // .catch(() => {
+  //   console.log('Is it here?');
+  // })
+
```

Comment:
I assume this is commented out as will only be used for seeding when necessary? A short comment explaining this would be a nice touch.